Created basic implementation of view classes.

Miracle2001 opened the pull request.

dylancoffey99 commented on system/views/view.py:
```diff
@@ -0,0 +1,84 @@
+
+
+class View(tk.Tk):
+
```
Remove blank line

dylancoffey99 replied:
Again, remove this blank line

dylancoffey99 commented on system/views/view.py:
```diff
@@ -0,0 +1,84 @@
+        frame.tkraise()
+
+
+class RegistrationPage(tk.Frame):
```
Not entirely sure if these classes should be broken down into multiple .py files. Leave it for now.

dylancoffey99 commented on system/views/view.py:
```diff
@@ -0,0 +1,84 @@
+    def __init__(self, *args, **kwargs):
+        tk.Tk.__init__(self,  *args, **kwargs)
+
+        # creating a container
```
Some of these comments may not be necessary, especially here where it's pretty obvious a container is being created.

nadinefrawley commented:
Looks like a good start. I agree with Dylan's points

dylancoffey99 commented on system/views/login_page.py:
```diff
@@ -0,0 +1,31 @@
+import tkinter as tk
+from tkinter import ttk
+import registration_page
+import user_page
```
These imports will give you an error because it needs to take the file structure into account. It should be formatted like this:
from system.views.filename import ClassName

Miracle2001 commented:
Implemented the admin and user view. Fixed the other views. I'm still some errors from pylint such as cyclic import, too many variables and duplicate code, please ignore them but the code runs fine. I'm going to look for solutions to my problems but we can work with this for now.

dylancoffey99 commented:
This branch is outdated from the master. You need to update your branch so it includes the latest changes from master.

Make sure master is up to date, and that you're on the "miracle-view" branch in Pycharm. Go to your project folder and open a Git bash terminal. Type "git rebase master", this will update your branch to include the latest changes from master, it shouldn't conflict with anything.

dylancoffey99 commented on system/views/admin_page.py:
```diff
@@ -0,0 +1,30 @@
+from tkinter import ttk
+
+
+class AdminPage(tk.Frame):
```
This should be called AdminView, same with LoginView, RegisterView etc. If you're wondering if its the reason why your imports aren't working, its not. I've started to get the same problem with completely different class files.

dylancoffey99 commented on system/views/registration_page.py:
```diff
@@ -0,0 +1,71 @@
+import system.views.user_page
+
+
+class RegistrationPage(tk.Frame):
```
Might be simpler to just call this RegisterView.

dylancoffey99 commented on system/views/view.py:
```diff
@@ -0,0 +1,62 @@
+from system.views.admin_page import AdminPage
+
+
+class View(tk.Tk):
```
Two things could change/happen here:
1. The implementation of this class is moved to the __ main __ file outside of the system. (you'll need to rebase your branch against the master to see this file)
2. The __ main __ file will call this view and it will stay here.

Now that I'm thinking about it, it would probably be better for the __main__ to call this view. Let me know if you don't understand.

dylancoffey99 commented on system/views/user_page.py:
```diff
@@ -0,0 +1,38 @@
+from tkinter import ttk
+
+
+class UserPage(tk.Frame):
```
This should be called CustomerView also.

dylancoffey99 commented:
I decided to help out with the GUI, it's just taking way too long it should have been done weeks ago. We need this finished and out of the way so we know 100% what were trying to implement. What I have there is pretty much what we set out in the beginning. It doesn't work right now as the controllers have to be implemented.
